src/Saxulum/Console/Provider/ConsoleProvider.php

```diff
@@ -32,6 +33,11 @@ public function register(Container $container)
 
         $container['console'] = function () use ($container) {
             $console = new ConsoleApplication($container);
+            
+            if (!is_null($container['console.dispatcher'])) {
```

Comment:
I would prefer the simpler solution, cause for most it will fit even without silex